jdaviz/configs/cubeviz/plugins/unified_slider/unified_slider.py

```diff
@@ -17,6 +17,7 @@ class UnifiedSlider(TemplateMixin):
     min_value = Float(0).tag(sync=True)
     max_value = Float(100).tag(sync=True)
     linked = Bool(True).tag(sync=True)
+    wait = Int(300).tag(sync=True)
```

Comment:
This is great and is working well, but I think 100 ms here for the throttle value strikes a better balance for responsiveness. @dtn5ah @javerbukh would one of you give this a try and see if you agree?

```suggestion
    wait = Int(100).tag(sync=True)
```

Reply:
Yes, 100 feels much smoother.